lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -1500,6 +1500,25 @@ spv::LoopControlMask getLoopControl(const BranchInst *Branch,
       // TODO Express partial unrolling in SPIRV.
       if (S == "llvm.loop.unroll.count" || S == "llvm.loop.unroll.full")
         return spv::LoopControlUnrollMask;
+      if (S == "llvm.loop.ivdep.enable")
```

Comment:
Again. Multiple llvm.loop.* metadata are possible. It should be capable to combine bits of loop control mask.

Reply:
Done